packages/fxa-settings/src/components/PageSettings/index.stories.tsx

```diff
@@ -3,70 +3,89 @@
 import AppLayout from '../AppLayout';
 import { isMobileDevice } from '../../lib/utilities';
 import { mockAppContext, mockEmail, MOCK_ACCOUNT } from '../../models/mocks';
 import { MOCK_SERVICES } from '../ConnectedServices/mocks';
 import { AppContext } from 'fxa-settings/src/models';
+import { MOCK_LINKED_ACCOUNTS } from '../LinkedAccounts/mocks';
+import { Meta } from '@storybook/react';
+
 
 const SERVICES_NON_MOBILE = MOCK_SERVICES.filter((d) => !isMobileDevice(d));
 
-storiesOf('Pages/Settings', module)
-  .addDecorator((getStory) => <LocationProvider>{getStory()}</LocationProvider>)
-  .add('cold start', () => (
-    <AppContext.Provider
-      value={mockAppContext({
-        account: {
-          ...MOCK_ACCOUNT,
-          displayName: null,
-          avatar: { id: null, url: null },
-          recoveryKey: false,
-          totp: { exists: false, verified: false },
-          attachedClients: SERVICES_NON_MOBILE,
-        } as any,
-      })}
-    >
-      <AppLayout>
-        <PageSettings />
-      </AppLayout>
-    </AppContext.Provider>
-  ))
-  .add('partially filled out', () => (
-    <AppContext.Provider
-      value={mockAppContext({
-        account: {
-          ...MOCK_ACCOUNT,
-          displayName: null,
-          totp: { exists: true, verified: false },
-          attachedClients: SERVICES_NON_MOBILE,
-        } as any,
-      })}
-    >
-      <AppLayout>
-        <PageSettings />
-      </AppLayout>
-    </AppContext.Provider>
-  ))
+export default {
+  title: 'Pages/Settings',
+  component: PageSettings,
+} as Meta;
+
+const coldStartAccount = {
+  ...MOCK_ACCOUNT,
+  displayName: null,
+  avatar: { id: null, url: null },
+  recoveryKey: false,
+  totp: { exists: false, verified: false },
+  attachedClients: [SERVICES_NON_MOBILE[0]],
+} as unknown as Account;
+
+const partiallyFilledOutAccount = {
+  ...MOCK_ACCOUNT,
+  displayName: null,
+  totp: { exists: true, verified: false },
+  attachedClients: SERVICES_NON_MOBILE,
+  linkedAccounts: MOCK_LINKED_ACCOUNTS,
+} as unknown as Account;
+
+const completelyFilledOutAccount = {
+  ...MOCK_ACCOUNT,
+  subscriptions: [{ created: 1, productName: 'x' }],
+  emails: [
+    mockEmail('johndope@example.com'),
+    mockEmail('johndope2@gmail.com', false),
+  ],
+  attachedClients: SERVICES_NON_MOBILE,
+  linkedAccounts: MOCK_LINKED_ACCOUNTS,
+}
+
+const storyWithContext = (
+  account: Partial<Account>,
+  storyName?: string,
+  config?: Config,
+) => {
+  const context = config
+    ? { account: account as Account, config: config }
+    : { account: account as Account };
+
+  const story = () => (
+    <LocationProvider>
+      <AppContext.Provider value={mockAppContext(context)}>
+        <AppLayout>
+          <PageSettings/>
+        </AppLayout>
+      </AppContext.Provider>
+    </LocationProvider>
+  );
+  if (storyName) story.storyName = storyName;
+  return story;
+};
+
+export const ColdStart = storyWithContext(
+  coldStartAccount,
+  'cold start',
```

Comment:
I don't mind either way, but you could take out the story names here. They'll just show up in SB as `Cold Start` vs `cold start`. 🤷‍♀️

Reply:
oh interesting hmmmmm. I liked seeing them in the example I was modeling this on just because it was a little more explicit/less mysterious, but I think that confusion will become less and less frequent the more of these that we transition. Good with going either way!